### DISCUSSION TITLE
Player Timeline - BTW:

### DISCUSSION THREAD

**Discussion by flowerhorse** [brunoherbelin/vimix#84] [Ideas]
Player Timeline - BTW:

Although Vimix does not process audio I would find it good if in the player-timeline also the waveform of the film audio would be represented, because the film sound, if available, usually also refers to the events in the film and from therefore certain effect inserts are clearly readable at the waveform.
For playback work with capturing I would also find it good if the wavestream was simply let through, so that it can be recorded with. Even if there are latencies, this simplifies the dubbing process a lot because there is already an audio track for orientation. In the window or configuration should be an option for dubbing audio while capturing and if you want to see the waveform in player. Or maybe better, I think:  If audio than waveform with option 'no dubbing while capturing' in player.

**Comment by brunoherbelin** [comment]
To partially answer this request (and related to #104), a beta implementation of audio support is now available in vimix (for Beta 7606baa20b1b8608c0d91494906bfab9e254af9c and above) : you can enable audio for each source individually, and recording also encodes audio in the video (either produced mix or a sound input). 

Displaying the waveform in the player is not planned for now (as I wouln't know how to do that) but I leave the possibility open.
